pkg/cloudprovider/providers/vsphere/vsphere_test.go

```diff
@@ -225,7 +225,7 @@ func TestVolumes(t *testing.T) {
 		t.Fatalf("Cannot create a new VMDK volume: %v", err)
 	}
 
-	diskID, err := vs.AttachDisk(volPath, "")
+	diskID, _, err := vs.AttachDisk(volPath, "")
```

Comment:
since it will not be used by DetachDisk directly , it has to be a blank identifier here